telebot/types.py

```diff
@@ -10479,4 +10479,111 @@ def de_json(cls, json_string):
+
+    Telegram documentation: https://core.telegram.org/bots/api#inputpaidmediaphoto
+
+    :param type: Type of the media, must be photo
+    :type type: :obj:`str`
+
+    :param media: File to send. Pass a file_id to send a file that exists on the Telegram servers (recommended), pass an HTTP URL for
+        Telegram to get a file from the Internet, or pass “attach://<file_attach_name>” to upload a new one using multipart/form-data
+        under <file_attach_name> name. More information on Sending Files »
+    :type media: :obj:`str`
+
+    :return: Instance of the class
+    :rtype: :class:`InputPaidMediaPhoto`
+    """
+
+    def __init__(self, media, **kwargs):
+        super().__init__(type='photo', media=media)
+    
+class InputPaidMediaVideo(InputPaidMedia):
+    """
+    The paid media to send is a video.
+
+    Telegram documentation: https://core.telegram.org/bots/api#inputpaidmediavideo
+
+    :param type: Type of the media, must be video
+    :type type: :obj:`str`
+
+    :param media: File to send. Pass a file_id to send a file that exists on the Telegram servers (recommended), pass an HTTP URL for
+        Telegram to get a file from the Internet, or pass “attach://<file_attach_name>” to upload a new one using multipart/form-data
+        under <file_attach_name> name. More information on Sending Files »
+    :type media: :obj:`str`
+
+    :param thumbnail: Optional. Thumbnail of the file sent; can be ignored if thumbnail generation for the file is supported server-side.
+        The thumbnail should be in JPEG format and less than 200 kB in size. A thumbnail's width and height should not exceed 320.
+        Ignored if the file is not uploaded using multipart/form-data. Thumbnails can't be reused and can be only uploaded as a new file,
+        so you can pass “attach://<file_attach_name>” if the thumbnail was uploaded using multipart/form-data under <file_attach_name>.
+        More information on Sending Files »
+    :type thumbnail: :class:`InputFile`
+
+    :param width: Optional. Video width
+    :type width: :obj:`int`
+
+    :param height: Optional. Video height
+    :type height: :obj:`int`
+
+    :param duration: Optional. Video duration in seconds
+    :type duration: :obj:`int`
+
+    :param supports_streaming: Optional. Pass True if the uploaded video is suitable for streaming
+    :type supports_streaming: :obj:`bool`
+
+    :return: Instance of the class
+    :rtype: :class:`InputPaidMediaVideo`
+
+    """
+
+    def __init__(self, media, thumbnail=None, width=None, height=None, duration=None, supports_streaming=None, **kwargs):
+        super().__init__(type='video', media=media)
+        self.thumbnail = thumbnail
+        self.width = width
+        self.height = height
+        self.duration = duration
+        self.supports_streaming = supports_streaming
+
+    def to_dict(self):
+        data = super().to_dict()
+        if self.thumbnail:
+            data['thumbnail'] = self.thumbnail.to_dict()
```

Comment:
Thumbnail is "InputFile or String"

Should be

if self.thumbnail:
            data['thumbnail'] = self.thumbnail